pkg/apis/serving/v1beta1/predictor.go

```diff
@@ -68,6 +72,18 @@ type PredictorSpec struct {
 	ComponentExtensionSpec `json:",inline"`
 }
 
+type StorageUri struct {
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	Uri string `json:"uri"`
+
+	// +kubebuilder:default="/mnt/models"
+	// +kubebuilder:validation:Pattern="^/.*"
+	// +kubebuilder:validation:MaxLength=255
+	// +optional
+	Path string `json:"path"`
```

Comment:
I think this is really mount path ? Let’s add comment explaining this field and consider rename to mountPath.